### PR TITLE
Add "disable-download-folder" setting

### PR DESCRIPTION
**Pull request by richlanc**
Add "disable-download-folder" rdp setting, the intention is to remove the \\download special case from being created.

### CODE REVIEW

File: src/protocols/rdp/channels/rdpdr/rdpdr-fs-messages-file-info.c [hunk 1]
```diff
@@ -165,9 +165,10 @@ void guac_rdpdr_fs_process_set_rename_info(guac_rdp_common_svc* svc,
             "destination_path=\"%s\"", __func__, iorequest->file_id,
             destination_path);
 
+    guac_rdp_fs* fs = device->data;
     /* If file moving to \Download folder, start stream, do not move */
     if (strncmp(destination_path, "\\Download\\", 10) == 0
-			&& !((guac_rdp_fs*) device->data)->disable_download) {
+			&& !(fs->disable_download || fs->disable_download_folder)){
```
Comment (SpeeddY) [inline src/protocols/rdp/channels/rdpdr/rdpdr-fs-messages-file-info.c:171]: so the existing setting "disable_download" simply means that files are not auto downloaded to the guac server when dropped in that folder?

**Comment by SpeeddY** [comment]
The download folder is correctly disabled for my guac connection to the xrdp host (browser https connection)
image

but it still seems to appear for my remote desktop connection
image

File: src/protocols/rdp/channels/rdpdr/rdpdr-fs-messages-file-info.c (hunk 1 above)
Comment (richlanc) [reply src/protocols/rdp/channels/rdpdr/rdpdr-fs-messages-file-info.c:171]:
```
If set to true downloads from the remote server to client (browser) will be disabled. This includes both downloads down via the hidden Guacamole menu, as well as using the special "Download" folder presented to the remote server. The default is false, which means that downloads will be allowed.

If file transfer is not enabled, this parameter is ignored.
```
https://guacamole.apache.org/doc/gug/configuring-guacamole.html

**Review by SpeeddY — APPROVED** [review]
looks good

**Pull request merged**